erpnext/stock/doctype/purchase_receipt/purchase_receipt.js

```diff
@@ -187,12 +187,6 @@ erpnext.stock.PurchaseReceiptController = class PurchaseReceiptController extend
 				}
 				cur_frm.add_custom_button(__('Retention Stock Entry'), this.make_retention_stock_entry, __('Create'));
 
-				if(!this.frm.doc.auto_repeat) {
-					cur_frm.add_custom_button(__('Subscription'), function() {
-						erpnext.utils.make_subscription(me.frm.doc.doctype, me.frm.doc.name)
```

Comment:
Snce we are removing it the util code should also go away? 

https://github.com/frappe/erpnext/blob/caad4537c5107dc5b675e05dcf037142bcfd8c37/erpnext/public/js/utils.js#L264-L276